### ISSUE TITLE
Convergence speed of the network?

### ISSUE THREAD

**Issue by twmht** [XiaXuehai/faceboxes#10]
I tried to train the network with WIDERFACE, But I found that it's difficult to converge.

after the several epochs, both the precision and recall are less than 1%, did you have the similar problem?

it seems that the paper work is trained at least 200 epochs, did you train such a long epochs?

**Comment by XiaXuehai** [comment]
I did not train with widerface. and the number of epochs is 300 which is same as the paper.
In the paper, the author did not mention  the mask. so I did not try it

**Comment by xtanitfy** [comment]
It seems like the weights have been finished.

**Comment by laidy3** [comment]
@twmht How about train from sratch?

**Comment by twmht** [comment]
@laidy3 

I trained from scatch, do I need to use a pretrained weight?

**Comment by laidy3** [comment]
@twmht I think it doesn’tneed. How about the preformence of traing from scatch?

**Comment by twmht** [comment]
@laidy3 

since the convergence is slow, So I stop training early at 1xx epochs. I forgot the performance, it's very low.

**Comment by twmht** [comment]
@laidy3 

Did you tried to train from scratch? how about the performance?

**Comment by laidy3** [comment]
@twmht 
Xavier init, 12 hours, batch size 128, 145 epochs, val loss= 5.22

**Comment by XiaXuehai** [comment]
![image](https://user-images.githubusercontent.com/11585321/50532263-e4c6c880-0b51-11e9-9770-c63d8ef0271f.png)
150-300 epoch of my training loss.

**Comment by lucasjinreal** [comment]
@XiaXuehai @laidy3  So, how does the inference result
